independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1062 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "Custom connector that connects to the Power BI Admin API (https://docs.microsoft.com/en-us/rest/api/power-bi/admin)",

```

Comment:
@paulculmsee Can you please provide valid description value for connector ?

Documentation - [Description of connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)